vnext/PropertySheets/Bundle.Common.targets

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
props are supposed to be Imported by the app project. The idea is in the app project you'd do

Import props

override/add your own properties

Import targets

Reply:
@asklar 
I moved some things around.  But I'm not 100% sure this is correct.  Should the BundleCommand be defined in the props file at all?  Or should it just be moved into the targets file?  The way it currently is, if you're overriding something like BundleEntryFile you have to do it before you import the props, which is not how you described it above.